src/generic/delete-modal/DeleteModal.jsx

```diff
@@ -35,16 +40,24 @@ const DeleteModal = ({
         </ActionRow>
       )}
     >
-      <p>{intl.formatMessage(messages.description, { category })}</p>
+      <p>{description || intl.formatMessage(messages.description, { category })}</p>
```

Comment:
[question]: Can we create a variable for this?

Reply:
Fixed